docs/Proposals.md

```diff
@@ -1,3 +1,25 @@
-# Working with Proposals
+# Working with Proposals and Schemes
 
-[Documentation Coming Soon]
+The ability to create proposals, vote one's reputation, and stake one's reputation and tokens on an outcome of the vote are primary functions of a DAO in the DAOstack ecosystem.  Proposals and how they distribute tokens and reputation are fundamental to promoting coherence and collaboration within and between DAOs in the DAOstack ecosystem.
```

Comment:
> The ability to create proposals, vote one's reputation, and stake one's reputation and tokens on an outcome of the vote are primary functions of a DAO in the DAOstack ecosystem.

This sentence is not clear and need to be revised.

>reputation, and stake

redundant comma.

> ...stake one's reputation and tokens on an outcome...

Should be : stake one's reputation or tokens on an outcome

>The ability to create proposals, vote one's reputation, and stake one's reputation and tokens on an outcome of the vote are primary functions of a DAO in the DAOstack ecosystem.

This is true only for GenesisProtocol and is not the primary functions of a DAO,

Reply:
Attempted to clarify the top-level statement.  I'll talk more about GenesisProtocol in the sections to come